reana_client/cli/files.py

```diff
@@ -82,7 +82,7 @@ def get_files(
     size,
     human_readable_or_raw,
 ):  # noqa: D301
-    """List workspace files.
+    r"""List workspace files.
```

Comment:
We cannot keep `r` unless we reformat the whole docstring, see what it would do to `reana-client ls --help`.